packages/lib/import-enex-md-gen.test.ts

```diff
@@ -108,6 +107,14 @@ describe('import-enex-md-gen', () => {
 		expect(all[0].size).toBe(0);
 	});
 
+	it('should handle tasks', async () => {
+		const filePath = `${enexSampleBaseDir}/tasks.enex`;
+		await importEnex('', filePath);
+		const expectedMd = await shim.fsDriver().readFile(`${enexSampleBaseDir}/tasks.md`);
+		const note: NoteEntity = (await Note.all())[0];
+		expect(note.body).toEqual(expectedMd);
+	});
+
```

Comment:
Why do you need a separate test here? Isn't it possible to use the existing pattern of putting an ENEX file and corresponding MD file in /enex_to_md?

Reply:
Those HTML files are simply the content of an ENEX file. You would copy all that's after `en-note` in the `<content>` tag and that will be your HTML file. The advantage of doing so is that we can more easily see the HTML code (which is in a CDATA block otherwise), and we have a standard way to process these tests. Would you be ok changing this? If something's not clear let me know

Reply:
I see.  Unfortunately, the Evernote tasks aren't in the CDATA block, they're tagged onto the end like the resources are.

Reply:
Oh that's right, I missed that. In that case you're right we indeed need to keep the whole ENEX file